Tests/KSCrash/KSCrashIdentifierTests.m

```diff
@@ -0,0 +1,32 @@
+#import <XCTest/XCTest.h>
+#import <string.h>
+#import "BSG_KSCrashIdentifier.h"
+
+@interface KSCrashIdentifierTests : XCTestCase
+@end
+
+@implementation KSCrashIdentifierTests
```

Comment:
Can we include a test for `bsg_kscrash_generate_report_path`?

Reply:
Yeah I should - it turned out to be a bit tricky to stub the file store but I've added a task to improve coverage here.